packages/std/src/math.rs

```diff
@@ -334,6 +335,18 @@ impl<'de> de::Visitor<'de> for Uint128Visitor {
     }
 }
 
+impl Sum for Uint128 {
+    fn sum<I: Iterator<Item = Self>>(iter: I) -> Self {
+        iter.fold(Uint128::zero(), ops::Add::add)
+    }
+}
+
+impl<'a> Sum<&'a Uint128> for Uint128 {
+    fn sum<I: Iterator<Item = &'a Self>>(iter: I) -> Self {
+        iter.fold(Uint128::zero(), |a, b| a + *b)
```

Comment:
This can be implemented using `ops::Add::add` as well if the Add trait is implemented for reference right hand sides (see https://github.com/CosmWasm/cosmwasm/pull/575)

Reply:
Nice PR and thanks for those updates